.github/SUPPORT.md

```diff
@@ -0,0 +1,15 @@
+# Getting Support as a Scrapy User
+
+This repository's "Issues" pane is not the place to request support
+or advice on using Scrapy. For that, we suggest joining the [Scrapy
+community][community] over at:
+
```

Comment:
Let me suggest:

"This repository's "Issues" pane used only for bug reports and feature
requests. If you have a "how to" or "is it possible" question or if you need
support/advice on using Scrapy, we suggest joining the [Scrapy
+community][community] over at: